lib/widgets/subscription_list.dart

```diff
@@ -189,8 +189,10 @@ class _SubscriptionList extends StatelessWidget {
       itemBuilder: (BuildContext context, int index) {
         final subscription = subscriptions[index];
         final unreadCount = unreadsModel!.countInChannel(subscription.streamId);
-        // TODO(#712): if stream muted, show a dot for unreads
-        return SubscriptionItem(subscription: subscription, unreadCount: unreadCount);
+        final showMutedUnreadBadge = unreadCount == 0 && unreadsModel!.countInChannelNarrow(subscription.streamId) > 0;
```

Comment:
nit: line too long, and important information after column 80
```suggestion
        final showMutedUnreadBadge = unreadCount == 0
          && unreadsModel!.countInChannelNarrow(subscription.streamId) > 0;
```